pkgs/servers/nextcloud/packages/generate.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -I nixpkgs=../../../.. -i bash -p nc4nix
+#! nix-shell -I nixpkgs=../../../.. -i bash -p nc4nix jq
```

Comment:
This should be a separate commit.

Reply:
Please don't drop this change. It should be made but just in a separate commit.

Reply:
It used to be, together with yet another commit that updated all packages, but rebasing became too annoying which is why I squashed it.